cmd/bad-key-revoker/main.go

```diff
@@ -254,13 +254,17 @@ func (bkr *badKeyRevoker) invoke() (bool, error) {
 		}
 		return false, err
 	}
+	bkr.logger.AuditInfo(fmt.Sprintf("found unchecked block key to work on: %v", unchecked))
```

Comment:
%#v will be more descriptive here.

Reply:
Actually you should probably just do this manually, since %v/%#v will format the key hash in a way that's impossible for a human to read anyway, `[revoked by %d, key hash %x]` would be much easier to scan.

Reply:
How about implementing `.String()` on the object so I can just use %s, since it's used in a few different log lines?